DataManagementSystem/Service/IRODSStorageElementHandler.py

```diff
@@ -579,7 +577,7 @@ def export_getResourceInfo( self , resource = None ):
     """ Send the storage element resource information
 """
 
-    conn , error, userDict = self.__irodsClient( )
+    conn , error, _ = self.__irodsClient()
```

Comment:
I tend to agree with Christophe, it is better to see what we actually omit in such case